### ISSUE TITLE
Keep consistent headings in admin section

### ISSUE THREAD

**Issue by rt4914** [forem/forem#18875]
In admin section we have headings like `Overview`, `Members `, `Spaces` and a lot more. Some of these headings are using `h1` and some are using `h2`. 
These should be kept consistent and we should be using `h1` at all places.

For example : `Response Templates` using `h2` but `Overview` uses `h1`.

<img width="1512" alt="Screenshot 2022-12-16 at 12 40 59 PM" src="https://user-images.githubusercontent.com/9396084/208042821-dc7f0dba-6d14-490f-8b12-ed82e83484a3.png">
<img width="1511" alt="Screenshot 2022-12-16 at 12 41 10 PM" src="https://user-images.githubusercontent.com/9396084/208042852-50838d5c-6e8b-4cfc-bcc3-3b31ab6d76d0.png">

### Task
- [ ] Use `h1` tags instead of `h2` for page headings as shown in above case
- [ ] Wrap `h1` instead a `header` tag if the `h1` has other views with it like a button, or link.

**Comment by maestromac** [comment]
Let's go with h1 because it is the first thing on the page.

From my code skimming, I think a header tag is only necessary when there's another component to the header area, like a button that needs to be displayed on the same level as the title.

**Comment by Ridhwana** [comment]
Trying to figure these rubocop errors out when making changes to the some of these files https://forem-team.slack.com/archives/C7GE84DEJ/p1673272052854839
